README.md

```diff
@@ -11,6 +11,23 @@
 
 <img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
 
+
+<!--<a name="handling-secrets"></a>-->
+
+<!-- You can find more documentation about JCasC here: -->
+
+- [Handling Secrets](./docs/features/secrets.adoc)
+- [Exporting configurations](./docs/features/configExport.md)
+- [Validating configurations](./docs/features/jsonSchema.md)
+- [Triggering Configuration Reload](./docs/features/configurationReload.md)
+- [JSON Schema documentation](./docs/features/jsonSchema.md)
```

Comment:
Actually remove this it’s covered 2 lines up sorry I missed before

Reply:
All the links?

Reply:
done